### ISSUE TITLE
Sent messages a bit hard to read

### ISSUE THREAD

**Issue by NPadrutt** [Odonno/Modern-Gitter#52]
Since the backgroudn color of the sent message bubble is a bit brigth in my opinion the text get's a bit hard to read. I guess if we make the color a tad darker this could be much improved.

![wp_ss_20151202_0002](https://cloud.githubusercontent.com/assets/1764367/11526561/8b4f1988-98db-11e5-848f-2b4b061d29cf.png)

**Comment by Odonno** [comment]
I do not understand. What color ?
You mean the command bar ?

**Comment by NPadrutt** [comment]
no, the background color of the message bubble together with the font color. light grey with white.
![wp_ss_20151202_0002](https://cloud.githubusercontent.com/assets/1764367/11529255/26a1aaee-98ec-11e5-8203-fe347ae70b81.png)

**Comment by Odonno** [comment]
Ok, you mean the background of your own messages.
I see. I let you try.
